src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Server/Properties/launchSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "useWebAssemblyDebugging": true
+}
```

Comment:
Can you start from an existing launchSettings template? This one is missing all the config, if you are adding it, keep it in sync with the rest :)

Reply:
Sure